metrics-new-relic/src/main/java/com/palominolabs/metrics/newrelic/NewRelicReporter.java

```diff
@@ -215,6 +226,7 @@ private void doGauge(String name, Gauge gauge) {
     }
 
     private void record(String name, float value) {
+        logger.trace("Reporting metric Custom/{} with value {}", name, value);
```

Comment:
`metricNamePrefix` should be included. Or better yet, calculate the metric name once, and use it in logging too. :)